### PR TITLE
Chapters 21 - 23

### PR DESCRIPTION
**Pull request by CodeWarrior-Hawaii**
OK, this is a big 'un.  I had forgotten that Chapter 22 was 42 million words long.  I have commits broken in to Punctuation, Callsigns, and Re-Linerating (undoing the forced word-wrapping).

Let me know what I missed because I am sure that I missed or screwed up something in this gigantic "Chapter".

### CODE REVIEW

**Comment by ctwelve** [comment]
Oh Goddamn this is a biggie!

**Review by ctwelve — CHANGES_REQUESTED** [review]
Hello. In the datelines you've deleted all trailing space. This is incorrect, because the (at least) two extra spaces after the dateline tell Markdown to do a linebreak without a new paragraph. Basically works like `<br>` does in HTML.

Please put them back, because @HamboneHFY considers the stacked datelines essential to the presentation.

File: content/books/deathworlders/chapter-22-warhorse-pt1-first-year.md [hunk 1]
```diff
@@ -1,928 +1,566 @@
----
-title: "Chapter 22: Warhorse Part 1—The First Year"
-date: 2015-09-18
-aliases:
-  - /book/deathworlders/chapter-22-warhorse
----
-
-**Date Point: 4y 10m AV**  
+--- title: "Chapter 22: Warhorse Part 1—The First Year" date: 2015-09-18 aliases:   - /book/deathworlders/chapter-22-warhorse ---
```
Comment (ctwelve) [inline content/books/deathworlders/chapter-22-warhorse-pt1-first-year.md]: The metadata block is...pretty thoroughly broken, in all the files. It's a YAML front matter block encased in three dashes. It has to be formatted **precisely**  and will not tolerate this.

File: content/books/deathworlders/chapter-22-warhorse-pt2-second-year.md [hunk 2]
```diff
@@ -1,366 +1,233 @@
----
-title: "Chapter 22: Warhorse Part 2—The Second Year"
-date: 2015-09-19
-aliases:
-  - /book/deathworlders/chapter-22-warhorse/second-year
----
-
-**Date Point: First Contact Day, 6y AV**  
+--- title: "Chapter 22: Warhorse Part 2—The Second Year" date: 2015-09-19 aliases:   - /book/deathworlders/chapter-22-warhorse/second-year ---
```
Comment (ctwelve) [inline content/books/deathworlders/chapter-22-warhorse-pt2-second-year.md]: Same deal here. Broken metadata! And...so on through all the files.

**Review by ctwelve — CHANGES_REQUESTED** [review]
Metadata! Also I note the ellipsis character has appeared; we don't want it! Please replace with three periods everywhere.

**Comment by CodeWarrior-Hawaii** [comment]
Alright.  Ellipsis character burninated with extreme prejudice.  Chapter metadata blocks fixed.  Spaces reverted to section headers.  Also some spelling and punctuation fixes.

**Review by ctwelve — APPROVED** [review]
Looks good!

**Pull request merged**